tests/convergence/test_bgp_dp_dp_convergence.py

```diff
@@ -12,29 +12,31 @@ def test_bgp_dp_dp_convergence(api, utils, bgp_convergence_config):
        dp/dp convergence
     """
 
-    api.set_config(bgp_convergence_config)
+    response = api.set_config(bgp_convergence_config)
+    assert(len(response.errors)) == 0
     # name of the port that should be shutdown to trigger withdraw route
     primary_rx_port = bgp_convergence_config.ports[1].name
 
     # Start traffic
```

Comment:
this script is for demonstration, so don't use utils to fetch stats. also, we shouldn't fetch "all" stats. only the ones we need.